executor-scripts/linux/static

```diff
@@ -20,7 +22,13 @@ addr_family() {
 configure_addresses() {
 	for i in $(ifquery -p address -i $INTERFACES_FILE $IFACE); do
 		addrfam=$(addr_family $i)
-		${MOCK} ip $addrfam addr $1 $i dev $IFACE
+		if [ "${IF_POINTOPOINT}" -a "${addrfam}" = "-4" ]; then
```

Comment:
`[ "${IF_POINTOPOINT}" -a "${addrfam}" = "-4" ] && PEER="peer ${IF_POINTOPOINT}"`

Reply:
This get's messy with the required else branch in a one-liner. I'd rather keep it that way.